src/Common/utils.ts

```diff
@@ -170,12 +171,12 @@ class Utils {
   /**
    * Check if the given string is valid hex.
    *
-   * @param input The input to check.
+   * @param input - The input to check.
    * @returns true if the input is valid hex, otherwise false.
    */
   public static isHex(input: string): boolean {
-    const hexRegEx = /([0-9]|[a-f])/gim
-    return (input.match(hexRegEx) || []).length === input.length
+    const hexRegEx = /([0-9]|[a-f])/gimu
```

Comment:
So, apparently, JS regexes have a bunch of wonky behavior unless you enable the `u` unicode flag:

https://eslint.org/docs/rules/require-unicode-regexp